applications/nrf_desktop/description.rst

```diff
@@ -0,0 +1,1082 @@
+
+   .. tab:: Short bypass board
+
+      .. figure:: /images/nrf_desktop_400398_debug.svg
+         :alt: nRF Desktop bypass board (short)
+
+      The shorter nRF desktop bypass board can be used with the desktop mouse.
+
+   .. tab:: Long bypass board
+
+      .. figure:: /images/nrf_desktop_400398_long_debug.svg
+         :alt: nRF Desktop bypass board (long)
+
+      The longer nRF desktop bypass board can be used with the gaming mouse.
+
+..
+
+Power management
+================
+
+Reducing power consumption is important for every battery-powered device.
+
+The nRF Desktop peripherals are either suspended or powered off when they are not in use for a defined amount of time:
+
+* In the suspended state, the device maintains the active connection.
+* In the powered off state, the CPU is switched to the off mode.
+
+In both cases, most of the functionalities are disabled.
+For example, LEDs are turned off and advertising is stopped.
+
+Moving the mouse or pressing any button wakes up the device and turns on the disabled functionalities.
+
+You can define the amount of time after which the peripherals are suspended or powered off using the :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` Kconfig option.
+By default, this period is set to 120 seconds.
+
+.. important::
+   When the gaming mouse is powered from USB, the power down timeout functionality is disabled.
+
+   If a nRF Desktop device supports remote wakeup, the USB connected device goes to suspended state when USB is suspended.
+   The device can then trigger remote wakeup of the connected host on user input.
+
+.. _nrf_desktop_configuration:
+
+Configuration
+*************
+
+|config|
+
+This section also describes the configuration sources that are used for the default configuration.
+
+Configuration sources
+=====================
+
+The nRF Desktop application uses the following files as configuration sources:
+
+* Devicetree Specification (DTS) files - These reflect the hardware configuration.
+  See :ref:`zephyr:dt-guide` for more information about the DTS data structure.
+* :file:`_def` files - These contain configuration arrays for the application modules and are specific to the nRF Desktop application.
+* Kconfig files - These reflect the software configuration.
+  See :ref:`kconfig_tips_and_tricks` for information about how to configure them.
+
+For information about differences between DTS and Kconfig, see :ref:`zephyr:dt_vs_kconfig`.
+
+The nRF Desktop introduces application-specific Kconfig options that can be used to simplify an application configuration.
+For more information, see the :ref:`nrf_desktop_application_Kconfig` page.
+
+You can also configure the following feature in the nRF Desktop application:
+
+* :ref:`Memory layout <nrf_desktop_memory_layout>`
+* :ref:`Bluetooth <nrf_desktop_bluetooth_guide>`
+* :ref:`Bootloader and Device Firmware Update <nrf_desktop_bootloader>`
+* :ref:`nrf_desktop_board_configuration`
+* :ref:`nrf_desktop_nrf21540ek`
```

Comment:
Maybe we could list those 2 above, next to paragraph describing `nrf_desktop_application_Kconfig` (those do not directly relate to configurable application's features).
E.g.:
```
The nRF Desktop introduces application-specific Kconfig options that can be used to simplify an application configuration.
For more information, see the :ref:`nrf_desktop_application_Kconfig` page.

The nRF Desktop application can be used with various hardware boards.
For more information about board support in the application, see :ref:`nrf_desktop_board_configuration`.

The nRF Desktop application can be used together with `nRF21540 EK` shield to benefit from an RF front-end module (FEM) for the 2.4 GHz range extension. 
For more information, see  :ref:`nrf_desktop_nrf21540ek`.
```